### PR TITLE
Univalence of two standard constructions

### PR DESCRIPTION
**Pull request by arnoudvanderleer**
Showing that the sigma displayed category is univalent when the two displayed categories that it is based on are univalent, and that a displayed category over unit is univalent when the category that it is based on is univalent

### CODE REVIEW

**Comment by peterlefanulumsdaine** [comment]
converting to draft since as I understand this is still WIP and not meant to be ready for review/merging yet

**Comment by benediktahrens** [comment]
@arnoudvanderleer : is this still WIP, or is the PR ready for review?

**Comment by arnoudvanderleer** [comment]
> @arnoudvanderleer : is this still WIP, or is the PR ready for review?

It is almost ready for review. Currently just redirecting references to sigma_disp_cat, and then it is ready for review.

File: UniMath/CategoryTheory/DisplayedCats/Examples/Sigma.v [hunk 1]
```diff
@@ -0,0 +1,294 @@
+Require Import UniMath.Foundations.All.
+Require Import UniMath.MoreFoundations.All.
+Require Import UniMath.CategoryTheory.Core.Categories.
+Require Import UniMath.CategoryTheory.Core.Isos.
+Require Import UniMath.CategoryTheory.Core.NaturalTransformations.
+Require Import UniMath.CategoryTheory.Core.Functors.
+Require Import UniMath.CategoryTheory.Core.Univalence.
+Require Import UniMath.CategoryTheory.Equivalences.Core.
+Require Import UniMath.CategoryTheory.Adjunctions.Core.
```
[the remark this answers is not in the document]
Comment (arnoudvanderleer) [reply UniMath/CategoryTheory/DisplayedCats/Examples/Sigma.v:9]: Oh, right, I forgot about that indeed. Will do!

**Comment by arnoudvanderleer** [comment]
> @arnoudvanderleer : a file in a satellite repository does not compile any more:
> 
> ```
>   File "./Satellite/TypeTheory/CwF_TypeCat/CwF_SplitTypeCat_Equiv_Cats.v", line 74, characters 5-19:
>   Error: Error: The reference sigma_disp_cat was not found in the current environment.
> ```
> 
> Is it correct that this simply needs a new `Require Import...` statement?

That is correct indeed.

**Pull request merged**